packages/desktopjs/src/notification.ts

```diff
@@ -37,11 +37,11 @@ export abstract class ContainerNotification {
      * @param {NotificationPermissionCallback} [callback] An optional callback function that is called with the permission value. Depcrecated in favor of the promise return value
      * @returns {Promise<string>} A Promise that resolves to the permission picked by the user.
      */
-    static requestPermission(callback?: NotificationPermissionCallback): Promise<string> { 
+    static async requestPermission(callback?: NotificationPermissionCallback): Promise<string> { 
```

Comment:
Wouldn't the return value have to change too?  Regardless, does this need to stay as before given it is an implementation of the w3c defined api - https://developer.mozilla.org/en-US/docs/Web/API/Notification/requestPermission ?

Reply:
Adding "async" doesn't change the signature, it still returns a Promise<string>, you just don't have to wrap it in a promise manually, the "async" does that automatically.

Reply:
I think I didn't change the public API, only on 1-2 protected functions (like onMoving in the snapping logic) where I made it async, so instead of "void" it now returns "Promise&lt;void&gt;".